test/fixtures/attrs-reflection-app.js

```diff
@@ -0,0 +1,21 @@
+import {Component, h} from '../../lib';
+
+export class AttrsReflectionApp extends Component {
+  static get attrsSchema() {
+    return {
+      'str-attr': {type: `string`},
+      'bool-attr': {type: `boolean`, default: true},
+      'number-attr': {type: `number`, default: 0},
+      'json-attr': {type: `json`},
+
+
```

Comment:
spaaace

Reply:
you want aligned key value pairs ? Not sure what you mean?

Reply:
i don't want these random two lines of whitespace